File: Teme.Payment.Data/DTO/PaymentDto.cs
using System;
using System.Collections.Generic;
using System.Text;
using Teme.Shared.Data.Context;
using Teme.Shared.Data.Primitives.Contract;

namespace Teme.Payment.Data.DTO
{
    public class PaymentDto
    {
        public int Id { get; set; }
        /// <summary>
        /// Id Договора
        /// </summary>
        public int ContractId { get; set; }

        public string ContractNumber { get; set; }
        public DateTime ContractDateCreate { get; set; }

        /// <summary>
        /// Тип договора
        /// </summary>
        public ContractFormEnum? ContractForm { get; set; }

        /// <summary>
        /// Номер регистрационного удостоверения
        /// </summary>
        public string CardNumber { get; set; }

        /// <summary>
        /// Тип ИМН/МТ
        /// </summary>
        public bool? IsTypeImnMt { get; set; }

        /// <summary>
        /// Торговое название
        /// </summary>
        public string TradeName { get; set; }

        public string NameRu { get; set; }
        public string NameKz { get; set; }

        /// <summary>
        /// Область применения на государственном языке
        /// </summary>
        public string ApplicationAreaKz { get; set; }

        /// <summary>
        /// Область применения на русском языке
        /// </summary>
        public string ApplicationAreaRu { get; set; }

        /// <summary>
        /// Назначение на государственном языке
        /// </summary>
        public string AppointmentKz { get; set; }

        /// <summary>
        /// Назначение на русском языке
        /// </summary>
        public string AppointmentRu { get; set; }

        /// <summary>
        /// Закрытая система
        /// </summary>
        public bool? IsClosedSystem { get; set; }

        /// <summary>
        /// Обоснование от производителя (указать страницу регистрационного досье)
        /// </summary>
        public string RationaleManufacturer { get; set; }

        /// <summary>
        /// Класс в зависимости от степени потенциального риска применения
        /// </summary>
        public int? DegreeRiskClassId { get; set; }

        /// <summary>
        /// Бланк
        /// </summary>
        public bool? IsBlank { get; set; }

        /// <summary>
        /// Средство измерения
        /// </summary>
        public bool? IsMeasures { get; set; }

        /// <summary>
        /// ИМН и МТ для ин витро диагностики
        /// </summary>
        public bool? IsDiagnostics { get; set; }

        /// <summary>
        /// Стирильное
        /// </summary>
        public bool? IsStyryl { get; set; }

        /// <summary>
        /// В наличие лекарственное средства
        /// </summary>
        public bool? IsPresenceMedicinalProduct { get; set; }

        /// <summary>
        /// Количество модификации ИМН
        /// </summary>
        public string NumberModificationImn { get; set; }

        /// <summary>
        /// Редакция до внесения изменений
        /// </summary>
        public string RevisionBeforeChanges { get; set; }

        /// <summary>
        /// Вносимые изменения
        /// </summary>
        public string ChangesMade { get; set; }

        public virtual IEnumerable<PaymentEquipmentDto> PaymentEquipmentDtos { get; set; }
        public virtual IEnumerable<PaymentPackagingDto> PaymentPackagingDtos { get; set; }
        public virtual IEnumerable<PaymentPlatformDto> PaymentPlatformDtos { get; set; }
    }
}

File: Teme.Web/src/app/ext/ext-payment/payment-dto.ts

export class PaymentDto {
  public id: number;
  public contractId: number;
  public contractNumber: string;
  public contractDateCreate: Date;
  public contractDateInterval: string;
  public contractForm: number;
  public cardNumber: string;
  public cardBeginDate: Date;
  public cardEndDate: Date;
  public isTypeImnMt: boolean;
  public tradeName: string;
  public nameRu: string;
  public nameKz: string;
  public applicationAreaKz: string;
  public applicationAreaRu: string;

  public appointmentKz: string;
  public appointmentRu: string;
  public isClosedSystem: boolean;
  public rationaleManufacturer: string;
  public degreeRiskClassId: number;
  public isBlank: boolean;

  public isMeasures: boolean;
  public isDiagnostics: boolean;
  public isStyryl: boolean;
  public isPresenceMedicinalProduct: boolean;
  public numberModificationImn: string;
  public revisionBeforeChanges: string;
  public changesMade: string;

  public paymentEquipmentDtos: Array<PaymentEquipmentDto>;
  public paymentPackagingDtos: Array<PaymentPackagingDto>;
  public paymentPlatformDtos: Array<PaymentPlatformDto>;
}
export class PaymentEquipmentDto {
  public id: number;
  public equipmentType: any;
  public name: string;
  public model: string;
  public manufacturer: string;
  public country: any;
}
export class PaymentPackagingDto {
  public id: number;
  public packagingType: any;
  public name: string;
  public sizeWidth: string;
  public sizeHeight: string;
  public sizeLength: string;
  public sizeMeasure: any;
  public volumeValue: string;
  public volumeMeasureId: number;
  public numberUnitsInBox: string;
  public shortDescription: string;
}
export class PaymentPlatformDto {
  public id: number;
  public countryId: number;
  public nameRu: string;
  public nameKz: string;
  public nameEn: string;
  public legalAddress: string;
  public factAddress: string;
}
